src/components/ChatLoader/index.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './styles.scss'
+
+export const ChatLoader = ({ height, width, color, dotRadius, dotPositions, className }) => {
+  return (
+    <svg height={height} width={width} className={className}>
+      {dotPositions.map((pos, index) => (
+        <circle
+          key={index}
+          className="dot"
+          cx={pos.cx}
+          cy={pos.cy}
+          r={dotRadius}
+          style={{ fill: color }}
+        />
```

Comment:
**Suggestion:** Use a more descriptive key for the map function instead of the index.
```suggestion
      {dotPositions.map((pos) => (
        <circle
          key={`${pos.cx}-${pos.cy}`}
          className="dot"
          cx={pos.cx}
          cy={pos.cy}
          r={dotRadius}
          style={{ fill: color }}
        />
      ))}
```